test/framework/fixtures.tmpl

```diff
@@ -0,0 +1,184 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: {{.Namespace}}
+resources:
+- {{ .FixtureDepth }}test/testdata/fixtures/{{ .Fixture }}
+
+patches:
+{{ if .K8ssandraCluster }}
+{{ with .K8ssandraCluster }}
+  - target:
+      kind: K8ssandraCluster
+    patch: |-
+
+      - op: replace
+        path: /spec/auth
+        value: {{ if .Auth }}{{ .Auth }}{{ else }}true{{ end }}
+
+  {{ if and $.Storage .Cassandra.StorageConfig }}
+      - op: replace
+        path: /spec/cassandra/storageConfig/cassandraDataVolumeClaimSpec/storageClassName
+        value: {{ $.Storage }}
+  {{ end }}
+
+  {{ if .Cassandra.Networking }}
+      - op: replace
+        path: /spec/cassandra/networking/hostNetwork
+        value: {{ $.HostNetwork }}
+  {{ end }}
+
+  {{if $.Zones }}
+    {{ range $rackIndex, $rack := .Cassandra.Racks }}
+      {{ if index $rack.NodeAffinityLabels "topology.kubernetes.io/zone" }}
+      - op: replace
+        path: /spec/cassandra/racks/{{ $rackIndex }}/nodeAffinityLabels/topology.kubernetes.io~1zone
```

Comment:
Is that a type in `topology.kubernetes.io~1zone` with the `~1`?

Reply:
Heh. That's the way of specifying a field containing a slash. According to http://jsonpatch.com/#json-pointer :

> If you need to refer to a key with ~ or / in its name, you must escape the characters with `~0` and `~1` respectively. For example, to get "baz" from `{ "foo/bar~": "baz" }` you’d use the pointer `/foo~1bar~0`.

Reply:
Neat :)